examples/flutter_gallery/lib/demo/rally/charts/vertical_fraction_bar.dart

```diff
@@ -0,0 +1,45 @@
+// Copyright 2019-present the Flutter authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
+
+class VerticalFractionBar extends StatelessWidget {
+  const VerticalFractionBar({this.color, this.fraction});
```

Comment:
const VerticalFractionBar( {this.color, this.fraction} );

Reply:
I put a space between the { } and the arguments. That's what you're referring to here? Seems to be a style inside of the flutter/flutter repo but I can't find the rules for this in the style guide.

Reply:
Yep that is style pattern for flutter/flutter